data/tlg0010/tlg011/__cts__.xml

```diff
@@ -5,10 +5,10 @@
       
    <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0010.tlg011.perseus-grc2" workUrn="urn:cts:greekLit:tlg0010.tlg011" xml:lang="grc">
         
-      <ti:label xml:lang="eng">Panegyricus</ti:label>
-        
-      <ti:description xml:lang="eng">Isocrates. Isocrates with an English Translation in three volumes, by George Norlin, Ph.D., LL.D. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd. 1928-1980.</ti:description>
+      <ti:label xml:lang="grc">Πανηγυρικός</ti:label>
         
+      <ti:description xml:lang="eng">Isocrates. Isocrates, Vol. 1. Norlin, George, editor. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd., 1928 (printing).</ti:description>
```

Comment:
I think we can go with  just `Isocrates, Vol. 1.` here.

Reply:
I think for the first batch, I have Isocrates. Isocrates, Vol. 1 ,  Isocrates. Isocrates, Vol. 2, etc. Would you mind approving this one @lcerrato and then I will do a batch revision of this with another pull request?

Reply:
@lcerrato actually it looks like an artifact of this batch,I will go change that for consistency.